residential.html

```diff
@@ -0,0 +1,1081 @@
+		pattern1 ... patern11	= pattern background
+		menu-vertical-hide		= hidden, open on click
+		
+		BACKGROUND IMAGE [together with .boxed class]
+		data-background="assets/images/_smarty/boxed_background/1.jpg"
+	-->
+
+<body class="smoothscroll enable-animation">
+
+
+	<!-- wrapper -->
+	<div id="wrapper">
+
+		<!-- 
+				AVAILABLE HEADER CLASSES
+
+				Default nav height: 96px
+				.header-md 		= 70px nav height
+				.header-sm 		= 60px nav height
+
+				.b-0 		= remove bottom border (only with transparent use)
+				.transparent	= transparent header
+				.translucent	= translucent header
+				.sticky			= sticky header
+				.static			= static header
+				.dark			= dark header
+				.bottom			= header on bottom
+				
+				shadow-before-1 = shadow 1 header top
+				shadow-after-1 	= shadow 1 header bottom
+				shadow-before-2 = shadow 2 header top
+				shadow-after-2 	= shadow 2 header bottom
+				shadow-before-3 = shadow 3 header top
+				shadow-after-3 	= shadow 3 header bottom
+
+				.clearfix		= required for mobile menu, do not remove!
+
+				Example Usage:  class="clearfix sticky header-sm transparent b-0"
+			-->
+		<div id="header" class="navbar-toggleable-md sticky clearfix">
+
+			<!-- TOP NAV -->
+			<header id="topNav">
+				<div class="container">
+
+					<!-- Mobile Menu Button -->
+					<button class="btn btn-mobile" data-toggle="collapse" data-target=".nav-main-collapse">
+						<i class="fa fa-bars"></i>
+					</button>
+
+					<!-- Logo -->
+					<a class="logo float-left" href="index.html">
+						<img src="assets/images/rocketElevators/rocketLogo.png" alt="" />
+					</a>
+
+					<!-- 
+							Top Nav 
+							
+							AVAILABLE CLASSES:
+							submenu-dark = dark sub menu
+						-->
+					<div class="navbar-collapse collapse float-right nav-main-collapse">
+						<nav class="nav-main">
+
+							<!-- 
+									.nav-onepage
+									Required for onepage navigation links
+									
+									Add .external for an external link!
+								-->
+							<ul id="topMain" class="nav nav-pills nav-main nav-onepage">
+								<li class="active"><!-- HOME -->
+									<a href="#home">
+										HOME
+									</a>
+								</li>
+								<li><!-- FEATURES -->
+									<a href="#features">
+										FEATURES
+									</a>
+								</li>
+								<li><!-- PRICING -->
+									<a href="#pricing">
+										PRICING
+									</a>
+								</li>
+								<li><!-- TESTIMONIALS -->
+									<a href="#testimonials">
+										TESTIMONIALS
+									</a>
+								</li>
+								<li><!-- PURCHASE -->
+									<a class="external" href="#purchase">
+										PURCHASE
+									</a>
+								</li>
+							</ul>
+
+						</nav>
+					</div>
+
+				</div>
+			</header>
+			<!-- /Top Nav -->
+
+		</div>
+
+
+		<!-- HOME -->
+		<section id="home" class="h-450" style="background:url('demo_files/images/wall2.jpg')">
+
+			<!-- TABLE -->
+			<div>
+				<h2 id="table-section-title">Top Rating Agent</h2>
+				<table>
+					<thead>
```

Comment:
This table needs more attribute information filled out. Not enough ids for css, nor is there buttons in the headers. They'll be useful for the js portion of this page.